.gitpod.yml

```diff
@@ -1,9 +1,11 @@
 ---
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - name: Install npm depdencies
-    init: npm i
+  - name: Install npm dependencies
+    init: |
+      npm ci
     command: |
+      npx playwright install chromium
```

Comment:
When running this command in the dockerfile, we shouldn't need it here

Reply:
Oh I see 👀 
An assumption from my side - I didn't test it. Could we potentially run `npx playwright install --with-deps chromium` in the dockerfile and it takes care of all?

Reply:
I found a way to install chromium in `node_modules` via the [Playwright docs](https://playwright.dev/docs/browsers#hermetic-install).

I updated the `.gitpod.yml` to do this in the init section.